draft-ietf-httpbis-binary-message.md

```diff
@@ -235,15 +234,14 @@ Other values cause the message to be invalid; see {{invalid}}.
 
 ## Request Control Data
 
-The control data for a request message includes four values that correspond to
-the values of the `:method`, `:scheme`, `:authority`, and `:path` pseudo-header
-fields described in HTTP/2 ({{Section 8.3.1 of H2}}). These fields are
-encoded, each with a length prefix, in the order listed.
+Request Control Data contains the method and target of a request message. That
```

Comment:
I notice now the capitalization here is incosistent with the rest of the body. I'll let you pick how to resolve that.